visual-testing-app/src/index.tsx

```diff
@@ -1,31 +1,15 @@
-import { useState } from 'react';
 import ReactDOM from 'react-dom';
 import {
   PortalsContainer,
   themesOverrides,
 } from '@commercetools-frontend/application-components';
 import { ThemeProvider } from '@commercetools-uikit/design-system';
 import Application from './application';
-import ThemeSwitcher, {
-  ThemeName,
-} from './components/theme-switcher/theme-switcher';
 
 const Main = () => {
-  const [selectedTheme, updateTheme] = useState<ThemeName>('default');
-  const isLocalEnvironment = process.env.NODE_ENV === 'development';
-
   return (
     <>
-      <ThemeProvider
-        theme={selectedTheme}
-        themeOverrides={themesOverrides[selectedTheme]}
-      />
-      {isLocalEnvironment ? (
-        <ThemeSwitcher
-          selectedTheme={selectedTheme}
-          onThemeChange={(newTheme) => updateTheme(newTheme)}
-        />
-      ) : null}
+      <ThemeProvider theme="test" themeOverrides={themesOverrides.default} />
```

Comment:
Shouldn't we remove the `theme=test` (assuming that we use the latest UI Kit version with the old theme being removed)?

Reply:
I decided to keep it like this so we don't have to depend on the new version of `ui-kit`.